src/Classes/AdldapUsers.php

```diff
@@ -456,6 +458,103 @@ public function password($username, $password, $isGUID = false)
     }
 
     /**
+     * Check if the php installation is able to perform a password change.
+     * Requires PHP 5.4 >= 5.4.26, PHP 5.5 >= 5.5.10 or PHP 5.6 >= 5.6.0
+     *
+     * @return bool
+     */
+    private function changePasswordSupported()
```

Comment:
This is all fantastic, but do you think this function would be better suited for the Ldap connection class since it's verifying if an LDAP function exists? Then we could just call `if ($this->connection->isBatchSupported())`?